examples/README.md

```diff
@@ -1,142 +1,6 @@
-- **MAPPO**
-    - *Feedforward*
-        - [decentralised][pz_mappo_ff_dec].
-
-### 2D RoboCup
-
-- **MAD4PG**:
-    a MAD4PG system running on the RoboCup environment.
-  - *Recurrent*
-    - [state_based][robocup_mad4pg_ff_state_based].
-
-## Discrete control
-
-We also include a number of systems running on discrete action space environments.
-
-### Debugging Environment - Simple Spread
-
-- **MAPPO**:
-      a MAPPO system running on the discrete action space simple_spread MPE environment.
-  - *Feedforward*
-    - [decentralised][debug_mappo_ff_dec] and [centralised][debug_mappo_ff_cen].
-
-- **MADQN**:
-      a MADQN system running on the discrete action space simple_spread MPE environment.
-  - *Feedforward*
-    - Decentralised
-        - [decentralised][debug_madqn_ff_dec]
-        - [decentralised lr scheduling][debug_madqn_ff_dec_lr_schedule] (***using lr schedule***)
-        - [decentralised custom lr scheduling][debug_madqn_ff_dec_custom_lr_schedule] (***using custom lr schedule***)
-        - [decentralised custom epsilon decay scheduling][debug_madqn_ff_dec_custom_eps_schedule] (***using configurable epsilon scheduling***).
-  - *Recurrent*
-    - [decentralised][debug_madqn_rec_dec].
-
-- **VDN**:
-      a VDN system running on the discrete action space simple_spread MPE environment.
-  - *Recurrent*
-    - [centralised][debug_vdn_rec_cen].
-
-### PettingZoo - Multi-Agent Atari
-
-- **MADQN**:
-   a MADQN system running on the two-player competitive Atari Pong environment.
-  - *Recurrent*
-    - [decentralised][pz_madqn_pong_rec_dec].
-
-- **MAPPO**:
-    a MAPPO system running on two-player cooperative Atari Pong.
-    - *feedforward*
-        - [decentralised][pz_mappo_coop_pong_ff_dec].
-
-### PettingZoo - Multi-Agent Particle Environment
-
-- **MADDPG**:
-      a MADDPG system running on the Simple Speaker Listener environment.
-  - *Feedforward*
-    - [decentralised][pz_maddpg_mpe_ssl_ff_dec].
-
-- **MADDPG**:
-      a MADDPG system running on the Simple Spread environment.
-  - *Feedforward*
-    - [decentralised][pz_maddpg_mpe_ss_ff_dec].
-
-### SMAC - StarCraft Multi-Agent Challenge
-
-- **MADQN**:
-    a MADQN system running on the SMAC environment.
-  - *Feedforward*
-    - [decentralised][smac_madqn_ff_dec].
-  - *Recurrent*
-    - [decentralised][smac_madqn_rec_dec].
-
-- **QMIX**:
-    a QMIX system running on the SMAC environment.
-  - *Recurrent*
-    - [centralised][smac_qmix_rec_cen].
-
-- **VDN**:
-    a VDN system running on the SMAC environment.
-  - *Recurrent*
-    - [centralised][smac_vdn_rec_cen].
-
-### OpenSpiel - Tic Tac Toe
-
-- **MADQN**:
-      a MADQN system running on the OpenSpiel environment.
-  - *Feedforward*
-    - [decentralised][openspiel_madqn_ff_dec].
+We include a non-exhaustive number of examples for Jax-based systems, showing common use-cases for Mava. We also have a [Jax quickstart notebook][jax_quickstart] that can be used to quickly create and train your first multi-agent system.
```

Comment:
We could probably remove Jax from the quickstart notebook link text.

Reply:
Done.

Reply:
Thanks